jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildTarTask.java

```diff
@@ -144,6 +144,9 @@ public void buildTar()
               projectProperties.getJavaLayerConfigurations().getLayerConfigurations(),
               helpfulSuggestions);
 
+      // TODO: This should not be called on projectProperties.
+      projectProperties.waitForLoggingThread();
```

Comment:
Is it worth waiting for the logging thread to complete? If it can take some time and we do wait for it, we add that amount of time to the build. Is it because finishing the log with an incomplete percentage may look weird?

But honestly, I do believe the wait time will be close to 0, so I'm actually fine with this. Just asking if you felt the strong need.

Reply:
When not waiting, last few lines get mixed up with whatever comes next in the build system build since the thread can keep running even after Jib finishes.